.editorconfig

```diff
@@ -10,5 +10,8 @@ insert_final_newline = true
 charset = utf-8
 end_of_line = lf
 
+[*.scss]
```

Comment:
Is there a reason to change this?

Reply:
yeah, it's more in line with "big" mature projects like CRA. 2 spaces feels more natural than 4 tbh after working with both